keyboards/meletrix/zoom75/info.json

```diff
@@ -0,0 +1,151 @@
+        { "label": "6", "x": 6, "y": 1.25 },
+        { "label": "7", "x": 7, "y": 1.25 },
+        { "label": "8", "x": 8, "y": 1.25 },
+        { "label": "9", "x": 9, "y": 1.25 },
+        { "label": "0", "x": 10, "y": 1.25 },
+        { "label": "-", "x": 11, "y": 1.25 },
+        { "label": "=", "x": 12, "y": 1.25 },
+        { "label": "Backspace", "x": 13, "y": 1.25 },
+        { "label": "Delect", "x": 15, "y": 1.25 },
+
+        { "label": "Tab", "x": 0, "y": 2.25, "w": 1.5 },
+        { "label": "Q", "x": 1.5, "y": 2.25 },
+        { "label": "W", "x": 2.5, "y": 2.25 },
+        { "label": "E", "x": 3.5, "y": 2.25 },
+        { "label": "R", "x": 4.5, "y": 2.25 },
+        { "label": "T", "x": 5.5, "y": 2.25 },
+        { "label": "Y", "x": 6.5, "y": 2.25 },
+        { "label": "U", "x": 7.5, "y": 2.25 },
+        { "label": "I", "x": 8.5, "y": 2.25 },
+        { "label": "O", "x": 9.5, "y": 2.25 },
+        { "label": "P", "x": 10.5, "y": 2.25 },
+        { "label": "[", "x": 11.5, "y": 2.25 },
+        { "label": "]", "x": 12.5, "y": 2.25 },
+        { "label": "\\", "x": 13.5, "y": 2.25, "w": 1.5 },
+        { "label": "PageUp", "x": 15, "y": 2.25 },
+
+        { "label": "Caps Lock", "x": 0, "y": 3.25, "w": 1.75 },
+        { "label": "A", "x": 1.75, "y": 3.25 },
+        { "label": "S", "x": 2.75, "y": 3.25 },
+        { "label": "D", "x": 3.75, "y": 3.25 },
+        { "label": "F", "x": 4.75, "y": 3.25 },
+        { "label": "G", "x": 5.75, "y": 3.25 },
+        { "label": "H", "x": 6.75, "y": 3.25 },
+        { "label": "J", "x": 7.75, "y": 3.25 },
+        { "label": "K", "x": 8.75, "y": 3.25 },
+        { "label": "L", "x": 9.75, "y": 3.25 },
+        { "label": ";", "x": 10.75, "y": 3.25 },
+        { "label": "'", "x": 11.75, "y": 3.25 },
+        { "label": "Enter", "x": 12.75, "y": 3.25, "w": 2.25 },
+        { "label": "Backspace1", "x": 14, "y": 1.25 },
+        { "label": "PageDown", "x": 15, "y": 3.25 },
+
+        { "label": "Shift1", "x": 0, "y": 4.25, "w": 1.25 },
+        { "label": "Shift2", "x": 1.25, "y": 4.25 },
+        { "label": "Z", "x": 2.25, "y": 4.25 },
+        { "label": "X", "x": 3.25, "y": 4.25 },
+        { "label": "C", "x": 4.25, "y": 4.25 },
+        { "label": "V", "x": 5.25, "y": 4.25 },
+        { "label": "B", "x": 6.25, "y": 4.25 },
+        { "label": "N", "x": 7.25, "y": 4.25 },
+        { "label": "M", "x": 8.25, "y": 4.25 },
+        { "label": ",", "x": 9.25, "y": 4.25 },
+        { "label": ".", "x": 10.25, "y": 4.25 },
+        { "label": "/", "x": 11.25, "y": 4.25 },
+        { "label": "Shift1", "x": 12.25, "y": 4.25, "w": 1.75 },
+        { "label": "\u2191", "x": 14, "y": 4.25 },
+
+        { "label": "Ctrl", "x": 0, "y": 5.25, "w": 1.25 },
+        { "label": "Win", "x": 1.25, "y": 5.25, "w": 1.25 },
+        { "label": "Alt", "x": 2.5, "y": 5.25, "w": 1.25 },
+
+        { "label": "EA", "x": 15.25, "y": 0 },
+        { "label": "EB", "x": 17.25, "y": 0 },
```

Comment:
```suggestion

```
As from elpekenin's suggestion about the removal of the bodge code for encoder rotations for VIA, these fake keys should be removed as well.